client/src/components/listbox/Listbox.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <Listbox 
+    :model-value="modelValue" 
```

Comment:
Würde ich abstrahieren in `value`, auch der emit würde ich vom `model` zu `update` ändern. Hintergrund ist der das wir versuchen alles über state transitions zu machen.